FAQ.md

```diff
@@ -16,9 +16,9 @@ You can turn any of your existing web apps into a module with a few extra lines
 * [COBI.bike system](https://get.cobi.bike)
 * Registration as a developer on [my.cobi.bike](https://my.cobi.bike)
 
-### Do I need to purchase a COBI.bike system to start building a module?
+### Do I need to purchase a COBI.Bike system to start building a module?
 
-No, you can get started for free with the [DevKit Simulator](https://github.com/cobi-bike/COBI.js-simulator) and test on a COBI.bike system when you're ready. If you don't have one yet, you can purchase a developer edition on [cobi.bike/devkit](https://cobi.bike/devkit) or any other COBI.bike system in our [online store](https://get.cobi.bike).
+No, you can get started for free with the [DevKit Simulator](https://github.com/cobi-bike/COBI.js-simulator) and test on a COBI.Bike system when you're ready. If you don't have one yet, you can purchase a developer edition on [cobi.bike/devkit](https://cobi.bike/devkit) or any other COBI.Bike system in our [online store](https://get.cobi.bike).
```

Comment:
the `DevKit Simulator` link is wrong

Reply:
Good point. I will change that. Funnily enough, GitHub knows how to redirect to the right repo

Reply:
yeah, but they only do it for a limited time. I think around 6 month or 1 year. However it will surely break in the future if we dont change it ;)